frontend/src/citizen-frontend/messages/MessagesPage.tsx

```diff
@@ -141,6 +145,17 @@ export default React.memo(function MessagesPage() {
                     onSend={(body) => sendMessage({ body })}
                     onSuccess={() => {
                       changeEditorVisibility(false)
+                      addTimedNotification({
+                        children: (
```

Comment:
Poista turh `<div>` wrapperi tästä
```suggestion
                        children: i18n.messages.messageEditor.messageSentNotification
```